Azure/azure-cli
# {App Service} Fix webapp tests

**Description**

Fix broken tests on `test_webapp_commands.py` .
 

**Testing Guide**

`azdev test_webapp_commands --live`.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
